### DISCUSSION TITLE
Cambier title en Edit, Create Resource y en ResourceManaager

### DISCUSSION THREAD

**Discussion by Vini-1975** [filamentphp/filament#14740] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

ambio title de resource por ejemplo  "Lider" en Edit y CreateResoruce Lider el titulo cambia a "lidere" como se puede hacer que eso cambie a "Lider" esto tambien en ResourceManager

**Comment by leandrocfe** [comment, marked as answer]
use

```php
return $table
    ->heading('Lider')
    ...
```

**Reply by Vini-1975** [reply]
Thank you so much, me puedes ayudar en otro problemita tengo una tabla miembros donde estan todos los registros del sistema existen 3 tipo de miembros(exploradore, lider, pastor) y se los registra segun su tipo para eso se creo registros para los miembros exploradores, lideres para los miembros lider y pastores para los miembros pastor, y una tabla premios, donde se tiene almacenado los diferentes tipos de premios que pueden tener, para registrar los premios que puede un miembro **registrados**, miembro **lideres** y miembro **pastores** se creo una tabla pivote premios_optenidos  para registrar los premios de cada uno de ellos pero cuando quiero crear una RelationManger LiderResource premiosoptenidos --generate tengo un problemas que cuando selecciono al explorador, lider, pastor en ves de mostra los premios que optubo me mustra todos los premioso de todos los miembros como puedo hacer esto

**Comment by Vini-1975** [comment]
Tengo estos filters en filament cuando selecciono un departamento el SectFilter provincia busca y coloca las provincias que pertencen al departamento y asi tambien para ciudad lo malo es que debo recargar la pagina para que los cambio se aplique a selectFilter provincia e igual para ciudad como puedo hacer que cuando se seleccione se actualice y no tenga que recargar la pagina manaulmente no tiene un comando como  ->reactive() o controle cuando se realice una seleccion este es el codigo "SelectFilter::make('ciudad_id')
                //->relationship('ciudad', 'nombre')
                ->options(function () {
                    $provincia = request()->get('tableFilters')['provincia_id'] ?? null;
                    if ($provincia) {
                        return Ciudade::where('provincia_id', $provincia)
                            ->pluck('nombre', 'id');
                    }
                    return Ciudade::all()->pluck('nombre', 'id');
                })
                ->query(function ($query, $state) {
                    if (is_array($state)) {                        
                        $state = implode(' ', $state);                        
                    } 
                    //dd($state);                  
                    if ($state <> '') {  
                        //dd($state,'2'); 
                        if ($state === '') {   
                            dd($state,'3');                         
                            $query->where('ciudad.id');
                            //dd('ddd');                            
                        }
                        elseif ($state <> null) {                           
                            //dd($state,'4');
                            $query->whereHas('ciudad', function ($q) use ($state) {
                                $q->where('id', $state);
                            });                            
                            //dd($state,'4',$query);
                        }                        
                    }                    
                })
                ->resetState(fn () => request()->query('tableFilters', ['departamento_id' => null, 'provincia_id' => null, 'ciudad_id' => null]))
                ,
            SelectFilter::make('provincia_id')
                //->relationship('ciudad.provincia', 'nombre')
                ->options(function () {
                    $departamentoId = request()->get('tableFilters')['departamento_id'] ?? null;
                    if ($departamentoId) {
                        return Provincia::where('departamento_id', $departamentoId)
                            ->pluck('nombre', 'id');
                    }
                    return Provincia::all()->pluck('nombre', 'id');
                })
                ->query(function ($query, $state) {
                    if (is_array($state)) {                        
                        $state = implode(' ', $state);                        
                    } 
                    //dd($state);                  
                    if ($state <> '') {  
                        //dd($state,'2'); 
                        if ($state === '') {   
                            dd($state,'3');                         
                            $query->where('ciudad.provincia.id');
                            //dd('ddd');                            
                        }
                        elseif ($state <> null) {                           
                            //dd($state,'4');
                            $query->whereHas('ciudad.provincia', function ($q) use ($state) {
                                $q->where('id', $state);
                            });                            
                            //dd($state,'4',$query);
                        }                        
                    }                    
                })
                ->resetState(fn () => request()->query('tableFilters', ['departamento_id' => null, 'provincia_id' => null, 'ciudad_id' => null]))
                
                ,
            SelectFilter::make('departamento_id')
                ->relationship('ciudad.provincia.departamento', 'nombre')                
                ->query(function ($query, $state) {
                    if (is_array($state)) {                        
                        $state = implode(' ', $state);                        
                    } 
                    //dd($state);                  
                    if ($state <> '') {  
                        //dd($state,'2'); 
                        if ($state === '') {   
                            dd($state,'3');                         
                            $query->where('ciudad.provincia.departamento.id');
                            //dd('ddd');                            
                        }
                        elseif ($state <> null) {                           
                            //dd($state,'4');
                            $query->whereHas('ciudad.provincia.departamento', function ($q) use ($state) {
                                $q->where('id', $state);
                            });                            
                            //dd($state,'4',$query);
                        }                        
                    }                    
                })
                
                ,      "
